pkgs/top-level/all-packages.nix

```diff
@@ -8966,7 +8966,7 @@ in
   swift = callPackage ../development/compilers/swift { };
 
   swiProlog = callPackage ../development/compilers/swi-prolog {
-    openssl = openssl_1_0_2;
+    openssl = openssl_1_1;
```

Comment:
`openssl` already points to `openssl_1_1`
```suggestion
```

Reply:
@marsam, sorry, I must be misunderstanding.  According to https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix#L8973 the swiprolog package is using openssl_1_0_2 as of this moment (2020 Mar 09 05:09 UTC, commit https://github.com/NixOS/nixpkgs/commit/f32e17d8025679535b4736e97c08a9112c0359a2).  Is this change existing elsewhere in a different PR that has not been merged yet?  Where is openssl already pointing to openssl_1_1?

Reply:
The suggestion means to use (default) `openssl` instead of explicitly specifying `openssl_1_1` (which is the default in 19.09 already).

Reply:
I'm not sure which of the two approaches is better; not a big deal, most likely.

Reply:
Ahh, thanks @vcunat, and sorry for my misunderstanding @marsam.  Yes, no need to override the default with ... the default.

Reply:
@kquick sorry, I should have been more clear